force-app/main/default/lwc/rd2EntryFormScheduleSection/rd2EntryFormScheduleSection.js

```diff
@@ -84,6 +95,31 @@ export default class rd2EntryFormScheduleSection extends LightningElement {
         }
     }
 
+    renderedCallback() {
+        this.applyCSSOnlyOnEperienceSite();
+    }
+
+    /***
+     * @description Applies CSS styles to rendered elements only for Experience Sites.
+     */
+    applyCSSOnlyOnEperienceSite() {
+        this.cssHideExperienceSite = this.isExperienceSite ? 'slds-hide' : '';
+        this.cssHideOnlyPaymentModal = this.isPaymentModal ? 'slds-hide' : '';
+        this.cssHideOnlyAmountFrequencyModal = this.isAmountFrequencyModal ? 'slds-hide' : '';
+        
+        
+        if(this.isExperienceSite) {
+            if(this.rd2State.dayOfMonth) {
+                this.isMonthlyDonation = !this.isElevateDonation ? true : false;
+            } else {
+                this.isMonthlyDonation = false;
+                let dd = String(new Date().getDate()).padStart(2, "0");
+                this.rd2State.dayOfMonth = dd === 31 ? LAST_DAY_OF_MONTH : dd;
```

Comment:
The last day of the month is not always "31". Plus, when Enhanced Recurring Donations is enabled we explicitly disable "31" as a picklist option for users. They can only select 1-30 and "Last Day of Month". Is there a reason why this is checking for 31?

Reply:
@force2b Hi Michael. You are right: the logic behind this is that, when the Recurring Donation is not monthly, and the user decides to change the installment period to "Monthly", today's date will be populated into the combobox for the user. Since, as you said, the value "31" is disabled for the DayOfMonth picklist, this checks that when the day is "31", the value shown in the combobox will never be "31", but will be "Last Day Of Month" instead. Does that make sense?

Reply:
It does. Thank you for the explanation.